src/couch_query_servers.erl

```diff
@@ -296,7 +296,8 @@ filter_docs(Req, Db, DDoc, FName, Docs) ->
     #httpd{} = HttpReq ->
         couch_httpd_external:json_req_obj(HttpReq, Db)
     end,
-    JsonDocs = [couch_doc:to_json_obj(Doc, [revs]) || Doc <- Docs],
+    Limit = config:get_integer("couchdb", "filter_revs_limit", 20),
```

Comment:
May be use 
```
Limit = config:get_integer("query_servers", "revs_limit", 20),
```

? Then we can reuse the same option for other design functions which receives _revisions list now.